### PR TITLE
Upgrade sonarqube to 8.9.1 version

### PR DESCRIPTION
**Pull request by lukasz-jarocki-sonarsource**

### CODE REVIEW

**Comment by lukasz-jarocki-sonarsource** [comment]
Hi @yosifkit . Could you let us know what is the issue here with the Munge PR / Diff Comment check?

File: library/sonarqube [hunk 1]
```diff
@@ -1,22 +1,22 @@
-Maintainers: Janos Gyerik <janos.gyerik@sonarsource.com> (@janos-ss),
-             Julien Lancelot <julien.lancelot@sonarsource.com> (@julienlancelot),
-             Christophe Levis <christophe.levis@sonarsource.com> (@christophelevis),
+Maintainers: Christophe Levis <christophe.levis@sonarsource.com> (@christophelevis),
              Pierre Guillot <pierre.guillot@sonarsource.com> (@pierre-guillot-sonarsource),
-             Michal Duda <michal.duda@sonarsource.com> (@michal-duda-sonarsource)
+             Tobias Trabelsi <tobias.trabelsi@sonarsource.com> (@tobias-trabelsi-sonarsource),
+             Lukasz Jarocki <lukasz.jarocki@sonarsource.com> (@lukasz-jarocki-sonarsource)
+
```
Comment (yosifkit) [inline library/sonarqube:5]: I think this new blank line is confusing the parser, so that it thinks the next block is not part of the "global" config.

**Comment by tianon** [comment]
(In the future, please force push to your existing PR branch and the PR will be automatically updated -- that will avoid spamming the maintainers with multiple PRs. :sweat_smile: :heart:)

File: library/sonarqube (hunk 1 above)
Comment (lukasz-jarocki-sonarsource) [reply library/sonarqube]: Thank you. It looks better now.

**Comment by lukasz-jarocki-sonarsource** [comment]
> 
> 
> (In the future, please force push to your existing PR branch and the PR will be automatically updated -- that will avoid spamming the maintainers with multiple PRs. 😅 ❤️)

I will, sorry for that.

File: library/sonarqube [hunk 2]
```diff
@@ -1,22 +1,21 @@
-Maintainers: Janos Gyerik <janos.gyerik@sonarsource.com> (@janos-ss),
-             Julien Lancelot <julien.lancelot@sonarsource.com> (@julienlancelot),
-             Christophe Levis <christophe.levis@sonarsource.com> (@christophelevis),
+Maintainers: Christophe Levis <christophe.levis@sonarsource.com> (@christophelevis),
              Pierre Guillot <pierre.guillot@sonarsource.com> (@pierre-guillot-sonarsource),
-             Michal Duda <michal.duda@sonarsource.com> (@michal-duda-sonarsource)
+             Tobias Trabelsi <tobias.trabelsi@sonarsource.com> (@tobias-trabelsi-sonarsource),
+             Lukasz Jarocki <lukasz.jarocki@sonarsource.com> (@lukasz-jarocki-sonarsource)
```
Comment (yosifkit) [inline library/sonarqube:4]: Welcome new maintainers! @tobias-trabelsi-sonarsource and @lukasz-jarocki-sonarsource. Thank you to the previous maintainers, @janos-ss, @julienlancelot, @michal-duda-sonarsource!

**Pull request merged**